monitoring/prometheus/etcdiscover/main.go

```diff
@@ -1,212 +0,0 @@
-// Copyright 2017 Google LLC. All Rights Reserved.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-// The etcdiscover binary monitors etcd to track the set of instances that
```

Comment:
I can't remember why this tool was [made](https://github.com/google/trillian/commit/a819abb06cec6881d4d4a57a837bf65ed6e99c53). Some CT operators use Prometheus (Let's Encrypt, as per #2105).
@pgporada Could you confirm you don't use it?

Reply:
We do not run `etcdiscover`. In our environment we have kubernetes prometheus-operator find everything with some specific etcd label. Thank you for asking and looking out for us!

Reply:
This seems promising then. I thought we might need this tool but was just deleting it to test whether the rest of the changes would fly first. If we don't know of any users, then maybe deleting it and then allowing it to be resurrected by someone with motivation & knowledge isn't a bad call.

@Martin2112 @AlCutter ?

Reply:
It was definitely used at one point to hook up monitoring. I thought it was used in our cloud deployment as well. Maybe things have changed under it and it no longer is.